### DISCUSSION TITLE
run-selenium-in-aws-lambda-for-ui-testing" ChromeDriver is assuming that Chrome has crashed

### DISCUSSION THREAD

**Discussion by vinitshetty** [CloudBytes-Academy/CloudBytes.dev#127] [General]
All steps succeeded but facing below error please help- 

https://cloudbytes.dev/snippets/run-selenium-in-aws-lambda-for-ui-testing

so ChromeDriver is assuming that Chrome has crashed

[ERROR] WebDriverException: Message: unknown error: Chrome failed to start: crashed.
  (chrome not reachable)
  (The process started from chrome location /opt/chrome/chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)

**Comment by jvillaweg** [comment]
same problem, were you able to succeed?

**Comment by vinitshetty** [comment]
Nope I gave up.

**Comment by rehanhaider** [comment]
@jvillaweg @vinitshetty The error is extremely generic and just means chrome hasn't been configured correctly. The exact reason is almost impossible to find out. 

More often than not it happens due to incorrect permission on the shell script.

**Comment by leiforion** [comment]
For those running into this issue, the latest version of Selenium is not compatible with the default code in the example. Use a version pre 4.0 and you will be good. You can update your requirements.txt to the following:

selenium>=3.0,<4.0

**Reply by rehanhaider** [reply]
Would you be willing to update the code?
